airflow/providers/google/marketing_platform/operators/campaign_manager.py

```diff
@@ -188,6 +194,11 @@ def __init__(
         self.profile_id = profile_id
         self.report_id = report_id
         self.file_id = file_id
+        if api_version in ["v3.4", "v3.5"]:
+            warnings.warn(
+                f"API {api_version} is deprecated and shortly will be removed. Please, consider using v4",
+                DeprecationWarning,
+            )
```

Comment:
This is redundant 20 April is just around the corner so these versions won't be useable any way.
I assume that if one submit `api_version`  to the API then the API itself will reject the request with relevant message? if so no need to add additional logic from Airflow side..

Reply:
Yes, for now if the user pass version 3.3 (that is currently deprecated), the error will show that there was no such version found for specific service.
Thanks for the idea, i will remove this check :)